docs/install-configuration.md

```diff
@@ -0,0 +1,175 @@
+
+!!! tip "Database addresses"
+    You can pass in multiple database addresses to Nakama via command like:
+
+    ```
+    nakama --database.address "root@db1:26257" --database.address "root@db2:26257"
+    ```
+
+### Runtime
+
+Options related to Lua-based runtime engine.
+
+| Parameter   | Flag               | Description
+| ---------   | ----               | -----------
+| `env`       | _N/A_              | List of Key-Value properties that are exposed to the Runtime scripts as environment variables.
+| `path`      | `runtime.path`     | Path of modules for the server to scan and load at startup. Default value is `data_dir/modules`.
+| `http_key`  | `runtime.http_key` | A key used to authenticate HTTP Runtime invocations. Default value is `defaultkey`.
+
+!!! warning "Important"
+    You must change `http_key` before going live with your app!
+
+### Socket
+
+Options related to connection socket and transport protocol between the server and clients.
+
+| Parameter                 | Flag                            | Description
+| ---------                 | ----                            | -----------
+| `server_key`              | `socket.server_key`             | Server key to use to establish a connection to the server. Default value is `defaultkey`.
+| `port`                    | `socket.port`                   | The port for accepting connections from the client, listening on all interfaces. Default value is 7350.
+| `max_message_size_bytes`  | `socket.max_message_size_bytes` | Maximum amount of data in bytes allowed to be read from the client socket per message. Default value is 1024.
+| `write_wait_ms`           | `socket.write_wait_ms`          | Time in milliseconds to wait for an ack from the client when writing data. Default value is 5000.
+| `pong_wait_ms`            | `socket.pong_wait_ms`           | Time in milliseconds to wait for a pong message from the client after sending a ping. Default value is 10000.
+| `ping_period_ms`          | `socket.ping_period_ms`         | Time in milliseconds to wait between client ping messages. This value must be less than the `pong_wait_ms`. Default value is 8000.
+
+!!! warning "Important"
+    You must change `server_key` before going live with your app!
+
+### Session
+
+You can change configuration options related to each user session, such as the encryption key used to create the token.
+
+| Parameter         | Flag                      | Description
+| ---------         | ----                      | -----------
+| `encryption_key`  | `session.encryption_key`  | The encryption key used to produce the client token. Default value is `defaultencryptionkey`.
+| `token_expiry_ms` | `session.token_expiry_ms` | Token expiry in milliseconds. Default value is 60000.
+
+!!! warning "Important"
+    You must change `encryption_key` before going live with your app!
+
+### Purchase
+
+Nakama can verify in-app purchases by connecting to various stores and keep a ledger of valid purchases.
```

Comment:
Could add a line there as well: "This is useful for preventing common in-app purchase replay attacks with valid receipts, as well as restoring purchases for user accounts as needed."